src/index.ts

```diff
@@ -125,14 +125,18 @@ export default (options: Options = {}): Plugin => {
           ? {}
           : await getOptions(dirname(id), options.tsconfig)
 
-      const result = await service.transform(code, {
+      const config = {
         loader,
         target: options.target || defaultOptions.target || 'es2017',
         jsxFactory: options.jsxFactory || defaultOptions.jsxFactory,
         jsxFragment: options.jsxFragment || defaultOptions.jsxFragment,
         define: options.define,
         sourcemap: options.sourceMap,
-      })
+      } as Record<string, unknown>
```

Comment:
can you change it to the proper type instead of `Record`?

Reply:
fixed!

Reply:
can you resolve the conflict?